openoa/utils/downloader.py

```diff
@@ -333,6 +336,248 @@ def get_era5_monthly(
     return df
 
 
+def get_era5_hourly(
+    lat: float,
+    lon: float,
+    save_pathname: str | Path,
+    save_filename: str,
+    start_date: str = "2000-01",
+    end_date: str = None,
+    calc_derived_vars: bool = False,
+) -> pd.DataFrame:
+    """
+    Get ERA5 data directly from the CDS service. This requires registration on the CDS service.
+    See registration details at: https://cds.climate.copernicus.eu/api-how-to
+
+    This function returns hourly ERA5 data from the "ERA5 monthly averaged data on single levels
+    from 1959 to present" dataset. See further details regarding the dataset at:
+    https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels-monthly-means.
+    U and V components of wind speed at 100 m, temperature at 2 m, and surface pressure are
+    downloaded here.
+
+    As well as returning the data as a dataframe, the data is also saved as monthly NetCDF files and
+    a csv file with the concatenated data. These are located in the "save_pathname" directory, with
+    "save_filename" prefix. This allows future loading without download from the CDS service.
```

Comment:
I was thinking about this, and I think we should just delete the monthly files, and save as one single netcdf and csv file as it's less onerous on the user who'll have to delete the originals if they're no longer required.